supervised-model-quality/script.whizzml

```diff
@@ -0,0 +1,51 @@
+(define SUPERVISED-MODELS ["model", "ensemble", "logistic-regression",
+                           "linear-regression", "deepnet"])
```

Comment:
Fusions were deliverately left out because, as they mix different models, there's no guarantee that they all have a unique dataset to start from. Thus, there's not a clear way of doing a train/test split for all of them in general.